tests/unit/backend/controllers/admin-forms.server.controller.spec.js

```diff
@@ -241,6 +241,51 @@ describe('Admin-Forms Controller', () => {
     })
   })
 
+  describe('transferOwner', () => {
+    it('should update admin id and set current owner as editor', async (done) => {
+      const collabAdmin = await User.create({
+        email: 'original@test.gov.sg',
+        _id: mongoose.Types.ObjectId('000000000002'),
+        agency: testAgency._id,
+      })
+
+      const newOwner = await User.create({
+        email: 'newOwner@test.gov.sg',
+        _id: mongoose.Types.ObjectId('000000000003'),
+        agency: testAgency._id,
+      })
+
+      const form1 = new Form({
+        title: 'Transfer Owner Form',
+        admin: collabAdmin._id,
+      })
+      await form1.save()
+
+      req.session.user = collabAdmin
+      req.body.email = newOwner.email
+      req.form = form1
+
+      res.json.and.callFake((args) => {
+        expect(args.form.admin._id.toString()).toEqual(newOwner._id.toString())
```

Comment:
Can we add another test that checks that the `newOwner` is removed from the `permissionList`?

Reply:
okay

Reply:
great!